### DISCUSSION TITLE
Django 3.2 support

### DISCUSSION THREAD

**Discussion by cyrus0824** [broadinstitute/seqr#4151] [General]
I noticed that Django 3.2 has recently (as of April 1, 2024) become unsupported:  https://www.djangoproject.com/download/

Any plans to upgrade to a newer LTS version, or something else?

**Comment by hanars** [comment]
Thanks for letting us know! We have created https://github.com/broadinstitute/seqr/issues/4152 to track the engineering work required for the upgrade

**Reply by cyrus0824** [reply]
Great!  Thank you so much.  I'll close this discussion and follow that issue.
